uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/ConfigurationsEditorEndpointTest.java

```diff
@@ -219,15 +186,12 @@ public void testEndpointAvailability() {
 	 */
 	@Test
 	public void testCreateEndpoint() {
-
-		try {
-			ConfigurationsEditor confEditor = new ConfigurationsEditor(PROJECT_NAME, CONTEXT);
-			confEditor.activate();
-			String[] title = element.split(" ");
-			confEditor.createNewGlobalEndpoint(title[0], element);
-			new DefaultTreeItem(new String[] { TYPE, title[0] + " (Endpoint)" }).select();
-		} catch (Exception e) {
-			fail(element);
+		createEndpoint();
+		String[] path = { ROOT, endpointName + " (Endpoint)" };
```

Comment:
The following piece of code is repeating many times in the code. Please extract it to a standalone method.
```
String[] path = { ROOT, endpointName + " (Endpoint)" };
WaitCondition wait = new TreeHasItem(new DefaultTree(), path);
new WaitUntil(wait, TimePeriod.MEDIUM, false);
if (wait.getResult() != null) {
	assertTrue("Tree does not contains Endpoint - " + endpointName, wait.getResult());
}
```